kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -323,13 +323,23 @@ define([
                 return false;
             }
             jobManager.doJobAction(action, [target]);
+
+            // if the job is being retried and the log viewer is open, close it
+            // TODO: a better fix!
```

Comment:
slightly hacky fix (but it works!) to stop the job log viewer from showing information about the original job when a retry is started

Reply:
Yeah, that seems to work. It's... a little hacky. Maybe in a future step, there's a way for the table to know it has a row open, and you could just ask the table? Anyway, fine for now.